traitsui/qt4/ui_panel.py

```diff
@@ -1349,11 +1349,13 @@ def __init__(self, parent, html_content, scale_dx, scale_dy):
 HeadingText = None
 
 
-def heading_text(*args, **kw):
+def heading_text(*args, create=False, **kw):
     """Create a Pyface HeadingText control."""
     global HeadingText
 
     if HeadingText is None:
         from pyface.api import HeadingText
 
-    return HeadingText(*args, **kw)
+    widget = HeadingText(create=create, *args, **kw)
```

Comment:
I think we need to make this change as well -

```suggestion
    widget = HeadingText(*args, create=create, **kw)
```